spec.md

```diff
@@ -3647,6 +3695,7 @@ primop = primop_2expr | primop_1expr | primop_1expr1int | primop_1expr2int ;
 (* Expression definitions *)
 expr =
     ( "UInt" | "SInt" ) , [ width ] , "(" , int_any , ")"
+  | type_enum , "(" , id , [ expr ] , ")"
```

Comment:
```suggestion
  | type_enum , "(" , id , [ "," , expr ] , ")"
```

Reply:
Thanks 😅 What is weird here is that bundle types are missing the `","`, but I do see it in most of the rest of the grammar (e.g. primops).